HEROKU.md

```diff
@@ -29,22 +29,33 @@ git push heroku main:master -f
 ```
 
 ### Step 3: Database Migration and Setup
-Before proceeding, please note that you can access your Heroku's command line by running the following from your machine:
+Next we need to set up the database and insert our institution in to the appropriate table.
+1. Access your Heroku instance by running
 ```
 heroku run bash
 ```
-Furthermore, you can access the Heroku Postgres database by running the following within that environment to execute SQL:
+2. If using Heroku Postgres, generate a migration by using:
 ```
-psql <the DATABASE_URL from your config vars>
+php bin/console doctrine:migrations:diff
```

Comment:
You probably need to remove this since the postgres migration stuff is in a separate PR